modules/keyboardshortcut/keyboardshortcut.js

```diff
@@ -1,91 +1,64 @@
-            function: function() {
-                JitsiMeetJS.analytics.sendEvent('shortcut.film.toggled');
-                APP.UI.toggleFilmStrip();
-            }
-        },
-        "M": {
-            character: "M",
-            id: "mutePopover",
-            function: function() {
-                JitsiMeetJS.analytics.sendEvent('shortcut.audiomute.toggled');
-                APP.conference.toggleAudioMuted();
-            }
-        },
-        "R": {
-            character: "R",
-            function: function() {
-                JitsiMeetJS.analytics.sendEvent('shortcut.raisedhand.toggled');
-                APP.conference.maybeToggleRaisedHand();
-            }
 
-        },
-        "T": {
-            character: "T",
-            function: function() {
-                JitsiMeetJS.analytics.sendEvent('shortcut.talk.clicked');
-                APP.conference.muteAudio(true);
-            }
-        },
-        "V": {
-            character: "V",
-            id: "toggleVideoPopover",
-            function: function() {
-                JitsiMeetJS.analytics.sendEvent('shortcut.videomute.toggled');
-                APP.conference.toggleVideoMuted();
-            }
-        },
-        "?": {
-            character: "?",
-            function: function(e) {
-                JitsiMeetJS.analytics.sendEvent('shortcut.shortcut.help');
-                APP.UI.toggleKeyboardShortcutsPanel();
-            }
-        }
-    };
+/**
+ * Initialise global shortcuts.
+ * Global shortcuts are shortcuts for features that don't have a button or
+ * link associated with the action. In other words they represent actions
+ * triggered _only_ with a shortcut.
+ */
+function initGlobalShortcuts() {
+
+    KeyboardShortcut.registerShortcut("ESCAPE", null, function() {
+        APP.UI.showKeyboardShortcutsPanel(false);
+    });
+
+    KeyboardShortcut.registerShortcut("?", null, function() {
+        JitsiMeetJS.analytics.sendEvent("shortcut.shortcut.help");
+        APP.UI.toggleKeyboardShortcutsPanel();
+    }, "keyboardShortcuts.toggleShortcuts");
+
+    KeyboardShortcut.registerShortcut("R", null, function() {
+        JitsiMeetJS.analytics.sendEvent("shortcut.raisedhand.toggled");
+        APP.conference.maybeToggleRaisedHand();
+    }, "keyboardShortcuts.raiseHand");
+
+    KeyboardShortcut.registerShortcut("T", null, function() {
+        JitsiMeetJS.analytics.sendEvent("shortcut.talk.clicked");
+        APP.conference.muteAudio(true);
+    }, "keyboardShortcuts.pushToTalk");
+
+    KeyboardShortcut.registerShortcut("F", 'filmstripPopover', function() {
+        JitsiMeetJS.analytics.sendEvent("shortcut.film.toggled");
+        APP.UI.toggleFilmStrip();
+    }, "keyboardShortcuts.toggleFilmstrip");
+
+    // Focus keys are directly implemented below.
+    KeyboardShortcut._addShortcutToHelp("0", "keyboardShortcuts.focusLocal");
+    KeyboardShortcut._addShortcutToHelp("1-9", "keyboardShortcuts.focusRemote");
```

Comment:
How are these shortcuts with the digits working? Maybe we should somehow register them from SmallVideo if it is possible? WDYT?

We can do this trough another PR if it is possible and if you like the idea.